src/listwiz/inspect.py

```diff
@@ -40,6 +40,28 @@ def longest_decreasing_streak(l):
     raise NotImplementedError
 
 
-def most_common_element(l):
-    # Find the most common element in the list
-    raise NotImplementedError
\ No newline at end of file
+def most_common_element(ll):
+    """Find the most common element in a list.
+
+    Parameters
+    ----------
+    ll : list
+
+    Returns
+    -------
+    most common element
+    """
+    
+    if not isinstance(ll, list):
+        raise ValueError("Input must be a list")
+    
+    counting ={}
+    for i in ll:
+        if i in counting:
+            counting[i] += 1
+        else:
+            counting[i] = 1
```

Comment:
You could also use `from collections import Counter` to avoid the `if` clause even.  But considering how basic the project, is I think this is a perfect solution.

Reply:
Thank you for the update. I was not aware of `Counter`.

```py
def most_common_element(ll):
   
    if not isinstance(ll, list):
        raise ValueError("Input must be a list")
    
    c = Counter(ll)
    return c.most_common(1)[0][0]
```

What do you mean by "likely have another iteration (for some other nitpicky/small thing)", please? Happy to learn.

Reply:
Sorry, I meant that it is not uncommon for someone to have small (often tiny/silly) asks to improve things a bit more.

Reply:
Happy to learn, thank you for the input :-)